doc/ruleSets/PER-CS1.0.rst

```diff
@@ -2,8 +2,6 @@
 Rule set ``@PER-CS1.0``
 =======================
 
-**This ruleset is deprecated** in favour of ``@PER-CS2.0``.
```

Comment:
I removed this because we should not deprecate `@PER-CS1.0` which is valid rule set, just like we don't like deprecate `@PSR1`,  `@PSR2` or  `@PSR12`.